feat(core): :sparkles: update code of CoreLinearProgress

PritamBag opened the pull request:
## Description
Update the code, where I implemented two props(showLabel, labelWidth), those props help display the label of linear progress, if showLabel is true then the label is shown, also labelWidth prop is used for the width of the label, by default it is 35px, if any users need more width or less they can customize it as well.
Ref: https://github.com/wrappid/core/issues/332 

## Related Issues

https://github.com/wrappid/native-web/issues/114

## Testing
I pass those props in the code of this(CoreLinearProgress.docs.js) file, and the code works properly.

## Checklist

- [X] I have performed a thorough self-review of my code.
- [ ] I have added or updated relevant tests for my changes.
- [X] My code follows the project's style guidelines and best practices.
- [X] I have updated the documentation if necessary.
- [ ] I have added or updated relevant comments in my code.
- [ ] I have resolved any merge conflicts in my branch.

## Screenshots (if applicable)
![image](https://github.com/user-attachments/assets/47dfa5f2-9566-49b2-8993-1c01f1f85723)

## Additional Notes

## Reviewers


## Maintainer Notes

- [ ] Has this change been tested in a staging environment?
- [ ] Does this change introduce any breaking changes or deprecations?

PritamBag committed:
feat(core): :sparkles: add prop

passing prop in LinearProgress, if the user passes this prop then the label is displayed in UI

Ref: #332

techoneel commented on package/components/feedback/CoreLinearProgress.js:
```diff
@@ -8,14 +8,15 @@ import { sanitizeComponentProps } from "../../utils/componentUtil";
       value={value}
       valueBuffer={valueBuffer}
       variant={variant}
+      showPercentage={showPercentage}
```
we must have show label property instead

techoneel reviewed (changes requested):
Do the changes needed.

PritamBag committed:
Merge branch 'development' of https://github.com/wrappid/core into WRPD-feature-332

PritamBag committed:
feat(core): :sparkles: update code of CoreLinearProgress

change code and add showLabel prop, also add labelWidth prop which has 35 value by default, but user can change it by passing this prop

Ref: #332

techoneel committed:
Merge branch 'development' into WRPD-feature-332

techoneel committed:
Merge branch 'development' into WRPD-feature-332

techoneel reviewed (approved):
good to go
